test/provisioner/provision.go

```diff
@@ -56,7 +56,7 @@ func NewCloudAPIAdaptor(provider string) (p *CloudAPIAdaptor) {
 	}
 }
 
-//GetCloudProvisioner returns a CloudProvision implementation
+// GetCloudProvisioner returns a CloudProvision implementation
 func GetCloudProvisioner(provider string, propertiesFile string) (CloudProvision, error) {
```

Comment:
@huoqifeng  Can you please rename `CloudProvision` to `CloudProvisioner` ?
This is a purely cosmetic change for easier readability. Can be done in a different PR but since you are creating the package, thought of suggesting it here.

Reply:
good point! thanks @bpradipt, done